src/openforms/contrib/customer_interactions/client.py

```diff
@@ -39,3 +45,42 @@ def get_digital_addresses_for_bsn(self, bsn: str) -> Iterator[DigitaalAdres]:
         }
         response = self.digitaal_adres.list_iter(params=params)
         yield from response
+
+    def create_customer_contact(
+        self, submission: Submission
+    ) -> MaakKlantContactResponse:
+        data: MaakKlantContactCreateData = {
+            "klantcontact": {
+                "kanaal": "open forms",
+                "onderwerp": submission.form.name,
+                "taal": "nld",
+                "vertrouwelijk": True,
+            },
+            "betrokkene": {
+                "rol": "klant",
+                "initiator": True,
+                "organisatienaam": "",
+            },
+            "onderwerpobject": {
+                "onderwerpobjectidentificator": {
+                    "objectId": submission.public_registration_reference,
+                    "codeObjecttype": "form",
+                    "codeRegister": "openforms",
+                    "codeSoortObjectId": "public_registration_reference",
+                }
+            },
+        }
+        return self.methods.maak_klant_contact(data=data)
+
+    def create_digital_address_for_betrokkene(
+        self, address, address_type, betrokkene_uuid
+    ):
+        data = DigitaalAdresCreateData(
+            adres=address,
+            soortDigitaalAdres=address_type,
+            omschrijving="Open Forms profile",
```

Comment:
do we need to provide an `omschrijving`? If yes, can we make it something like:


```suggestion
            omschrijving="Onderdeel van klantprofiel in webformulier",
```